frontend/components/WifiDetails.jsx

```diff
@@ -68,14 +69,19 @@ export const WifiDetails = ({ wifiNetwork }) => {
             <Typography>{wifiNetwork.BSSID}</Typography>
             <div className={classes.container}>
               <Typography component="div">
-                {wifiNetwork.SECURITY === 'WPA1 WPA2' ? (
+                <div className={classes.row}>
+                  {wifiNetwork.SECURITY.split(' ').map(security => (
+                    <Chip className={classes.chip} label={security} color="primary" />
+                  ))}
+                </div>
+                {/* {wifiNetwork.SECURITY === 'WPA1 WPA2' ? (
```

Comment:
unnecessary comment

Reply:
I'm sorry this slipped by. I will resolve it.